chapters/en/chapter1/3.mdx

```diff
@@ -67,20 +69,45 @@ There are three main steps involved when you pass some text to a pipeline:
 2. The preprocessed inputs are passed to the model.
 3. The predictions of the model are post-processed, so you can make sense of them.
 
+## Available pipelines for different modalities
```

Comment:
Nice! I like this section, I wonder if it makes sense to only list a few pipelines (most used ones, `text-generation`, `image-text-to-text`, etc) per modality and then link to documentation page for more.